### PR TITLE
sci-astronomy/stellarium: bump version to 0.19.0

### PR DESCRIPTION
**Pull request by DarthGandalf**
Signed-off-by: Alexey Sokolov <sokolov@google.com>
Closes: https://bugs.gentoo.org/684060
Package-Manager: Portage-2.3.62, Repoman-2.3.11

### CODE REVIEW

File: sci-astronomy/stellarium/stellarium-0.19.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils desktop flag-o-matic gnome2-utils virtualx
```
Comment (a17r) [inline sci-astronomy/stellarium/stellarium-0.19.0.ebuild]: EAPI-7 bump looks easy on this one and `gnome2-utils` should be dropped in favor of `xdg-utils` (same line count) or `xdg` (drops need for postinst and postrm, but needs to make sure that `src_prepare` running `cmake-utils_src_prepare` isn't dropped.

File: sci-astronomy/stellarium/stellarium-0.19.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	dev-qt/qtwidgets:5
+	media-fonts/dejavu
+	sys-libs/zlib
+	virtual/opengl
+	gps? ( dev-qt/qtpositioning:5 )
+	media? ( dev-qt/qtmultimedia:5[widgets] )
+"
+DEPEND="${RDEPEND}
+	dev-qt/qtconcurrent:5
+	nls? ( dev-qt/linguist-tools:5 )
```
Comment (a17r) [inline sci-astronomy/stellarium/stellarium-0.19.0.ebuild]: `nls` would be moved out into `BDEPEND` for EAPI-7.

File: sci-astronomy/stellarium/stellarium-0.19.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+src_test() {
+	virtx cmake-utils_src_test
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	# use the more up-to-date system fonts
+	rm "${ED%/}"/usr/share/stellarium/data/DejaVuSans{Mono,}.ttf || die
```
Comment (a17r) [inline sci-astronomy/stellarium/stellarium-0.19.0.ebuild]: In EAPI-7, `ED` would not be slash terminated anymore.

**Comment by a17r** [comment]
ping ;)

**Comment by DarthGandalf** [comment]
Yes, I'm still on it :)

When I tried EAPI=7, it stopped building due to link error, which looks related to --as-needed; I started to prepare a patch to fix it, but then I got stuck travelling for a while. Now I'm back and expect to fix it this week.

**Comment by a17r** [comment]
`EAPI-7`, cmake-utils makes the build default to `-DBUILD_SHARED_LIBS=ON`

**Comment by DarthGandalf** [comment]
Done.

**Pull request closed without merging**